README.md

```diff
@@ -21,6 +22,26 @@ can be used to resolve [phyloreferences](http://phyloref.org):
   `input.owl` by comparing their resolution with the expected resolution recorded
   in the file.
 
+# Build and execution instructions
+
+JPhyloRef can be built from source by running `mvn package` from the source directory.
+This will create a JAR file in the target directory, which can be executed by running,
```

Comment:
if you mean the directory named `target/`, I suggest to indicate this using trailing slash and backtics for formatting.

Reply:
Yes, that is definitely a lot clearer. Fixed in 2be5f0a.